src/basic/process-util.c

```diff
@@ -187,7 +167,6 @@ int get_process_cmdline(pid_t pid, size_t max_columns, ProcessCmdlineFlags flags
                 if (r < 0)
                         return r;
 
-                mfree(t);
```

Comment:
this should be kept really. otherwise you'll leak the empty buffer that read_full_virtual_file() returned.

Hmm, and maybe use plain free() for freeing it. After all mfree()'s only raison d'etre is that it returns NULL, so that you can do `p = mfree(p);`. But if that's not done here, free() is equally good

Reply:
Oh you're right. Forgot to re-add this after some logic changes.